halo2_proofs/src/plonk/circuit.rs

```diff
@@ -1,21 +1,23 @@
-use super::{lookup, permutation, Assigned, Error};
-use crate::dev::metadata;
-use crate::{
-    circuit::{Layouter, Region, Value},
-    poly::Rotation,
-};
 use core::cmp::max;
 use core::ops::{Add, Mul};
 use ff::Field;
 use sealed::SealedPhase;
 use std::cmp::Ordering;
 use std::collections::HashMap;
+use std::env::var;
 use std::fmt::{Debug, Formatter};
 use std::{
     convert::TryFrom,
     ops::{Neg, Sub},
 };
 
+use super::{lookup, permutation, Assigned, Error};
+use crate::dev::metadata;
+use crate::{
+    circuit::{Layouter, Region, Value},
+    poly::Rotation,
+};
+
```

Comment:
Unnecessary diff I think.

Reply:
This one either I think.

Reply:
Not sure what happened here.  Should be reverted now.